### DISCUSSION TITLE
Diving Headlight

### DISCUSSION THREAD

**Discussion by AFK-TankyMOFO** [FakeFishGames/Barotrauma#8522] [Ideas and suggestions]
Tilting the character's head downward a little bit when in water so the headlight aligns with our line of fire would be pretty nice.

At least do it when we are actively aiming for god's sake.

**Comment by someguythatplaysbarotrauma** [comment]
or they could just add attachable flashlights to guns

**Reply by Orum** [reply]
Fortunately there's a mod that does just that.  Unfortunately it doesn't require batteries, or work with all guns (even if you only consider vanilla guns), but it's better than nothing.

**Reply by leonard84** [reply]
IMHO Gunlights should be part of vanilla, with the janky way the character chooses to move, e.g., opening a door with the crowbar behind the back, the fixed light is pretty much unusable. The suite light also doesn't require any power 🤷.

**Comment by itchyOwl** [comment]
Implemented in the current unstable. Also there's a ticket about the flashlight attachment in our private repo.
